itest/lnd_multi-hop_force_close_test.go

```diff
@@ -780,8 +782,7 @@ func runMultiHopReceiverPreimageClaim(ht *lntest.HarnessTest,
 	if ht.IsNeutrinoBackend() {
 		numSweeps = 2
 	}
-
-	flakeSkipPendingSweepsCheckDarwin(ht, carol, numSweeps)
+	ht.AssertNumPendingSweeps(carol, numSweeps)
```

Comment:
Similar to the above case, Carol would need to make sure the HTLC has locked in at L737 so there's no pending remote commit. In addition, we assert the HTLC is still there when she settles it with the preimage. However, this `AssertNumActiveHtlcs` only asserts that the HTLC is still there, it doesn't assert the HTLC is now settled with a preimage, which means the new commitment dance may still be in the process while we now check her sweep requests. This happens because, previously we would create another pending remote commit for this settlement, although the link is already broken. This behavior is now fixed in the previous commit - we skip it when the link shuts down.